### PR TITLE
Title and logo are updated

### PR DESCRIPTION
**Pull request by developer-irfan**
Title of the website and logo are updated as you required. 

Screen shot:-

### CODE REVIEW

**Comment by warmachine028** [comment]
Could you please complete the PR Template. You can refer to other closed PRs on how to fill the template.

File: client/public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>UEMCRP (All Caps)</title>
```
Comment (warmachine028) [inline client/public/index.html:27]: (All Caps) was written for reference purpose, no need to add that in the title

**Comment by developer-irfan** [comment]
@warmachine028 ok i gona delete All caps

**Comment by warmachine028** [comment]
> @warmachine028 ok i gona delete All caps

Yes please make the changes

**Comment by warmachine028** [comment]
Hey @developer-irfan . What's the status of the work?

**Pull request merged**